src/Microsoft.Health.Dicom.Api/Features/Context/DicomRequestContextMiddleware.cs

```diff
@@ -45,6 +45,7 @@ public async Task Invoke(HttpContext context, IDicomRequestContextAccessor dicom
                 method: request.Method,
                 uri,
                 baseUri,
+                correlationId: System.Diagnostics.Activity.Current?.RootId,
```

Comment:
What is this value? How is it set?

Reply:
Each request received by the application is treated as Diagnostic Activity and is assigned Activity ID for debugging purposes. 
I read about it from the following sources:
- https://docs.microsoft.com/en-us/dotnet/core/diagnostics/distributed-tracing
- https://docs.microsoft.com/en-us/dotnet/core/diagnostics/distributed-tracing-concepts

I also confirmed that the correlation id now correlates with the request id captured in LinuxRequestTelemetry table by application insights logs.